internal/application/services/distributionLogUpdatedEventScanner.go

```diff
@@ -70,6 +74,17 @@ func (ds *DistributionLogUpdatedEventScanner) runScan(ctx context.Context) {
 		return
 	}
 
+	// Skip if tx receipt not found (nil). This means that the node does not store log receipts and there are no logs at all
+	receipt, err := ds.executionPort.GetTransactionReceipt(ds.csModuleTxReceipt)
```

Comment:
I would abstract this querying into another function that just returns true or false, so it can be used both here and in validator exit requests. That way it would be tidier and we could make it to be more robust in determining status. At first look it may look unusual why that exact tx receipt is being queried.

Also, we shouldn't assume how different execution clients respond and we need to test that. We know that Reth responses with `result: null` but we need to see how Erigon responds and handle it appropriately.